OptimizelySDKCore/OptimizelySDKCore/OPTLYEventBuilder.h

```diff
@@ -34,39 +34,45 @@ NS_ASSUME_NONNULL_END
 /**
  * Create the parameters for an impression event.
  *
- * @param config The project config object.
  * @param userId The ID of the user.
- * @param experimentKey The experiment name.
- * @param variationId The variation ID.
+ * @param experiment The experiment.
+ * @param variation The variation.
  * @param attributes A map of attribute names to current user attribute values.
  * @return A map of parameters for an impression event. This value can be nil.
  *
  */
-- (nullable NSDictionary *)buildImpressionEventTicket:(nonnull OPTLYProjectConfig *)config
-                                         userId:(nonnull NSString *)userId
-                                  experimentKey:(nonnull NSString *)experimentKey
-                                    variationId:(nonnull NSString *)variationId
-                                     attributes:(nullable NSDictionary<NSString *, NSString *> *)attributes;
+- (nullable NSDictionary *)buildImpressionEventTicketForUser:(nonnull NSString *)userId
```

Comment:
Why did you rename it? 
What's the purpose to use Ticket in the name?

Reply:
Since I made ProjectConfig as EventBuilder property to avoid passing in every method, so I had to rename h

Reply:
Since we are renaming, let's remove `ticket` from the name as none of the other SDKs call it that :)